andela/hc-bulls
# #156687693 Configure Periodic Reports

#### What does this PR do 
 Enables the user  to configure periodic reports, for example, daily, weekly and monthly

#### Description of Task to be completed
As a user I want to be able to decide when to receive the reports, whether daily, weekly or monthly
As a user I want a view of my reports on the dashboard

#### How should this be manually tested?
After logging in and setting up your e-mail in the Heroku Link, before setting up check you can access the Check report link under your account setting where you will not find any report on the dashboard, but after setting checks you will see the reports for the set checks.

You could also select daily reports as the option you want the you will receive an email for the daily reports.
  
#### What are the relevant pivotal tracker stories?
Link to the story : https://www.pivotaltracker.com/n/projects/2163622
Story ID : #156687693

#### Screenshots
This screenshot shows the dashboard added options for the user to configure periodic reports they prefer:
<img width="1274" alt="screen shot 2018-05-10 at 12 39 59 pm" src="https://user-images.githubusercontent.com/32554603/39863699-15b09ab6-5450-11e8-92c5-3adf6444c4b4.png">
This screenshot shows the dashboard with reports for user's checks:
<img width="1273" alt="screen shot 2018-05-10 at 12 47 58 pm" src="https://user-images.githubusercontent.com/32554603/39863843-7c8b2288-5450-11e8-9538-76a0cc2fa2fd.png">
